docs/Project.toml

```diff
@@ -1,5 +1,3 @@
 [deps]
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
-
-[compat]
-Documenter = "~0.22"
```

Comment:
Might want to keep this, just bump it to the current version.

Reply:
I think we don't need this `[compat]` table not to repeat the issue.
And, many other repositories also don't have this table.

* https://github.com/JuliaLang/julia/blob/master/doc/Project.toml
* https://github.com/JuliaPlots/PlotDocs.jl/blob/master/docs/Project.toml
* https://github.com/JuliaGraphics/Luxor.jl/blob/master/docs/Project.toml

Reply:
Those packages are out of luck when Documenter 0.26 is released. It is best practice to have it.

Reply:
I see. thanks for the information!